React to appfwk configuration classes no longer being singletons but …

eflumerf opened the pull request:
…paramters of init method

eflumerf committed:
React to appfwk configuration classes no longer being singletons but paramters of init method

eflumerf commented:
This should not be merged until after https://github.com/DUNE-DAQ/appfwk/pull/267

gcrone reviewed (approved):
All looks good to me.

eflumerf committed:
Remove DAQModuleManager

jcfreeman2 committed:
syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
Date:   Fri Sep 8 12:15:30 2023 -0500

    syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
    Date:   Tue Aug 29 17:25:57 2023 -0500

        syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
        Date:   Tue Aug 29 16:30:46 2023 -0500

            syncing dunedaq-develop-cpp-ci.yml; previous commit: Merge: f21e155 c947b06
            Author: eflumerf <61473357+eflumerf@users.noreply.github.com>
            Date:   Mon Jul 3 07:41:31 2023 -0500

                Merge pull request #51 from DUNE-DAQ/eflumerf/VersionBump

                Update version number

jcfreeman2 committed:
syncing ; previous commit: Author: John Freeman <jcfree@fnal.gov>
Date:   Tue Dec 12 19:37:54 2023 -0600

    syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
    Date:   Fri Sep 8 12:15:30 2023 -0500

        syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
        Date:   Tue Aug 29 17:25:57 2023 -0500

            syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
            Date:   Tue Aug 29 16:30:46 2023 -0500

                syncing dunedaq-develop-cpp-ci.yml; previous commit: Merge: f21e155 c947b06
                Author: eflumerf <61473357+eflumerf@users.noreply.github.com>
                Date:   Mon Jul 3 07:41:31 2023 -0500

                    Merge pull request #51 from DUNE-DAQ/eflumerf/VersionBump

                    Update version number

eflumerf committed:
Use datatype_to_string in code to avoid confusion about what string to use

jcfreeman2 committed:
syncing ; previous commit: Author: John Freeman <jcfree@fnal.gov>
Date:   Fri Dec 15 18:07:22 2023 -0600

    syncing ; previous commit: Author: John Freeman <jcfree@fnal.gov>
    Date:   Tue Dec 12 19:37:54 2023 -0600

        syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
        Date:   Fri Sep 8 12:15:30 2023 -0500

            syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
            Date:   Tue Aug 29 17:25:57 2023 -0500

                syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
                Date:   Tue Aug 29 16:30:46 2023 -0500

                    syncing dunedaq-develop-cpp-ci.yml; previous commit: Merge: f21e155 c947b06
                    Author: eflumerf <61473357+eflumerf@users.noreply.github.com>
                    Date:   Mon Jul 3 07:41:31 2023 -0500

                        Merge pull request #51 from DUNE-DAQ/eflumerf/VersionBump

                        Update version number

jcfreeman2 committed:
syncing dunedaq-develop-cpp-ci.yml; previous commit: Author: John Freeman <jcfree@fnal.gov>
Date:   Mon Jan 8 17:40:22 2024 -0600

    syncing ; previous commit: Author: John Freeman <jcfree@fnal.gov>
    Date:   Fri Dec 15 18:07:22 2023 -0600

        syncing ; previous commit: Author: John Freeman <jcfree@fnal.gov>
        Date:   Tue Dec 12 19:37:54 2023 -0600

            syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
            Date:   Fri Sep 8 12:15:30 2023 -0500

                syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
                Date:   Tue Aug 29 17:25:57 2023 -0500

                    syncing ; previous commit: Author: Pengfei Ding <9438483+dingp@users.noreply.github.com>
                    Date:   Tue Aug 29 16:30:46 2023 -0500

                        syncing dunedaq-develop-cpp-ci.yml; previous commit: Merge: f21e155 c947b06
                        Author: eflumerf <61473357+eflumerf@users.noreply.github.com>
                        Date:   Mon Jul 3 07:41:31 2023 -0500

                            Merge pull request #51 from DUNE-DAQ/eflumerf/VersionBump

                            Update version number

eflumerf committed:
Merge branch 'develop' into eflumerf/NoOKSSingletons
